kubernetes/apps/web3/monero/dashboard/resources/server.py

```
@@ -221,20 +259,54 @@ def proxy_observer_api(self):
     try:
-        with open(LOG_FILE) as f:
+        with open(path) as f:
             data = json.load(f)
-        with log_lock:
-            for k in log:
-                log[k] = deque(data.get(k, []))
-        print(f"Loaded {len(log['timestamps'])} old log entries")
+        for k in target:
+            target[k] = deque(data.get(k, []))
+        print(f"Loaded {len(target['timestamps'])} entries from {os.path.basename(path)}")
     except Exception as e:
```

Ruff (0.16.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)